modules/application-health-configuring.adoc

```diff
@@ -35,42 +35,48 @@ spec:
       tcpSocket:  <4>
         port: 8080 <5>
       initialDelaySeconds: 15 <6>
-      timeoutSeconds: 1 <7>
-    readinessProbe: <8>
-      httpGet: <9>
-        host: my-host <10>
-        scheme: HTTPS <11>
+      periodSeconds: 20 <7>
+      timeoutSeconds: 10 <8>
+    readinessProbe: <9>
+      httpGet: <10>
+        host: my-host <11>
+        scheme: HTTPS <12>
         path: /healthz
-        port: 8080 <12>
-    startupProbe: <13>
-      exec: <14>
-        command: <15>
+        port: 8080 <13>
+    startupProbe: <14>
+      exec: <15>
+        command: <16>
         - cat
         - /tmp/healthy
-      failureThreshold: 30 <16>
-      periodSeconds: 10 <17>
+      failureThreshold: 30 <17>
+      periodSeconds: 20 <18>
+      timeoutSeconds: 10 <19>
 ----
 <1> Specify the container name.
 <2> Specify the container image to deploy.
 <3> Optional: Create a Liveness probe.
 <4> Specify a test to perform, here a TCP Socket test.
 <5> Specify the port on which the container is listening.
 <6> Specify the time, in seconds, after the container starts before the probe can be scheduled.
-<7> Specify the number of seconds between probes.
-<8> Optional: Create a Readiness probe.
-<9> Specify the type of test to perform, here an HTTP test.
-<10> Specify a host IP address. When `host` is not defined, the `PodIP` is used.
-<11> Specify `HTTP` or `HTTPS`. When `scheme` is not defined, the `HTTP` scheme is used.
-<12> Specify the port on which the container is listening.
-<13> Optional: Create a Startup probe.
-<14> Specify the type of test to perform, here an Container Execution probe.
-<15> Specify the commands to execute on the container.
-<16> Specify the number of times to try the probe after a failure.
-<17> Specify the number of seconds to perform the probe.
+<7> Specify the number of seconds to perform the probe. The default is `10`. Must be greater than `timeoutSeconds`.
+<8> Specify the number of seconds of inactivity after which the probe is assumed to have failed. The default is `1`. Must be lower than `periodSeconds`.
```

Comment:
<9> to <17> the text did not change, only bumped the number.
<18> Modified the text (was <17>.
<19> New text